src/entity/EnemyShipSet.java

```diff
@@ -0,0 +1,140 @@
+package entity;
+
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+import screen.Screen;
+import engine.Cooldown;
+import engine.Core;
+import engine.DrawManager;
+import engine.DrawManager.SpriteType;
+import engine.GameSettings;
+import java.util.logging.Logger;
+
+public class EnemyShipSet {
+
+    // 생성된 적들을 관리하기 위한 Set
+    private Set<EnemyShip> enemies;
+    // 적 생성 쿨타임
+    private Cooldown spawnCooldown;
+    // DrawManager 인스턴스
+    private DrawManager drawManager;
+    // 게임 화면 정보
+    private Screen screen;
+    // 적 랜덤 생성을 위한 랜덤 객체
+    private Random random;
+    // 아군 함선 참조를 위한 변수
+    private Ship ship;
+    // 적 함선의 이동속도
+    private int movementSpeed;
+    // 적 함선의 X 방향 속도
+    private int X_speed = 1;
+    // 적 함선의 Y 방향 속도
+    private int Y_speed = 1;
```

Comment:
```suggestion
    // 적 함선의 X 방향 속도
    private double X_SPEED = 1.0;
    // 적 함선의 Y 방향 속도
    private double Y_SPEED = 1.0;
```
int 값으로 속도를 조절하니까 속도 변동이 너무 크더라구요

어차피 X_SPEED, Y_SPEED에 double형이 곱해져서 double형이 되니 애초에 SPEED를 double로 사용하면 미세한 속도 조절이 가능할 것 같습니다

Reply:
그래도 괜찮을 것 같습니다. 일단 기본속도로 1로 해둔거라서 이후에 변경점에 따라 변경해도 좋을 것 같아요